c/camel-quarkus/build-info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "camel-quarkus",
+    "github_url":"https://github.com/apache/camel",
```

Comment:
here @shreyakajbaje you will need to mention package url we are interested to build as result here `https://github.com/apache/camel-quarkus`